### PR TITLE
refactor(frontend): merge sendWizardSteps

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We merge `sendWizardSteps` and `icSendWizardSteps` into  a single one already being used after PR #1907.

### CODE REVIEW

File: src/frontend/src/lib/config/send.config.ts [hunk 1]
```diff
@@ -18,9 +13,21 @@ export const sendWizardSteps = (i18n: I18n): WizardSteps => [
 	{
 		name: WizardStepsSend.SENDING,
 		title: i18n.send.text.sending
-	},
+	}
+];
+
+export const sendWizardStepsWithQrCodeScan = (i18n: I18n): WizardSteps => [
+	...sendWizardSteps(i18n),
```
Comment (peterpeterparker) [inline src/frontend/src/lib/config/send.config.ts:20]: Why no TOKENS_LIST here?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/config/send.config.ts:20]: this is used by the `EthSendModal` and the `IcSendModal`, so we would need an object that has all steps except the tokens list. We can invert and create one that has all and the other one that filters out the TOKENS_LIST step

Comment (peterpeterparker) [reply src/frontend/src/lib/config/send.config.ts:20]: Ahhh now I get it. All good.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**